packages/backup-format/src/utils/backupPreview.ts

```diff
@@ -10,9 +10,10 @@ export interface BackupSummary {
     files: number
     wallets: string[]
     createdAt: number
+    countOfSmartPay?: number
 }
 
-export function getBackupSummary(json: NormalizedBackup.Data): BackupSummary {
+export function getBackupSummary(json: NormalizedBackup.Data, countOfSmartPay?: number): BackupSummary {
```

Comment:
Add `countOfSmartPay` into `NormalizedBackup.Data` and log more information about SmartPay wallets, not the count only.